tools/block-generator/generator/server.go

```diff
@@ -20,10 +20,12 @@ import (
 	"fmt"
 	"net/http"
 	"os"
+	"strconv"
 	"strings"
 	"time"
 
 	"github.com/algorand/go-algorand/tools/block-generator/util"
+	"github.com/gorilla/mux"
```

Comment:
Are you moving to Gorilla for easier route handling? Or other reasons?

Reply:
for easier route handling.